docs/hosting/installation/docker.md

```diff
@@ -7,12 +7,15 @@ contentType: tutorial
 
 [Docker](https://www.docker.com/){:target=_blank .external-link} offers the following advantages:
 
-* Install n8n in a clean environment.
+* Installs n8n in a clean environment.
 * Easier setup for your preferred database.
 * Can avoid issues due to different operating systems, as Docker provides a consistent system.
+* Can avoid compatibility issues due to differences in operating systems and tools.
```

Comment:
Is portability a thing worth mentioning? Makes it easier to migrate to a different host